.github/workflows/main.yml

```
@@ -0,0 +1,36 @@
+on: [push, pull_request]
+name: Test Code
+jobs:
+  test_it:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+        node_version:
+          - 10
+          - 12
+          - 14
+        architecture:
+          - x64
+        # an extra windows-x86 run:
+        include:
+          - os: windows-2016
```

pls change `windows-2016` to `windows-latest`
```suggestion
          - os: windows-latest
```